android/app/src/main/kotlin/ca/bigpanther/shipanther/Application.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.notificationTest
+import io.flutter.app.FlutterApplication
+import io.flutter.plugin.common.PluginRegistry
+import io.flutter.plugin.common.PluginRegistry.PluginRegistrantCallback
+import io.flutter.plugins.GeneratedPluginRegistrant
+import io.flutter.plugins.firebasemessaging.FlutterFirebaseMessagingService
+import io.flutter.plugins.firebasemessaging.FirebaseMessagingPlugin
+class Application:FlutterApplication(), PluginRegistrantCallback {
```

Comment:
AndroidManifest.xml should be modified with `<application android:name=".Application"`

Reply:
Added this in the new commits